advocacy_docs/partner_docs/Cyberark/05-Usingcyberark.mdx

```diff
@@ -0,0 +1,109 @@
+
+3. Select Databases —> PostgreSQL, and then click Edit. The platform parameters appear.
+
+![Select Platform](Images/platfom.png)
+
+4. In the Automatic Password Management —> General parameters Section, make the following change.
+       
+       - Set the ImmediateInterval parameter to 1. (The number of minutes that will elapse between when the user initiates an account management process and when the process is performed.)
+      
+![Select Platform](Images/passwordpolicy.png)
+
+5. Click Apply to save the changes, and then click OK.
+
+6. Restart the CPM (Central Policy Manager) Service named "CyberArk Password Manager".
+
+## Create the Account for EDB Postgres Advanced Server running instance
+
+The following steps demonstrate creating the Account on PVWA (Password Vault Web Access) for the EDB Postgres Advanced Server running instance, the password will be managed for this EDB Postgres Advanced Server running instance.
+
+1. Login to PVWA (Password Vault Web Access).
+
+2. Open Accounts page, click Add Account.
+
+3. On select system type screen, select database.
+
+![Select Platform](Images/database.png)
+
+4. On asssign to platform screen, select PostgreSQL.
+
+![Select Platform](Images/pg.png)
+
+5. On store in safe screen, select the safe to store the password.
+
+![Select Platform](Images/safe.png)
+
+6. On define properties screen, provide the connection information for the EDB Postgres Advanced Server running instance.
+
+![Select Platform](Images/conn1.png)
+
+![Select Platform](Images/conn2.png)
+
+7. Click on Add button, the account will be created.
+
+![Select Platform](Images/account.png)
+
+## Change the Password 
+
+The following steps demonstrate changing the password of EDB Postgres Advanced Server running instance.
+
+1. Login to PVWA (Password Vault Web Access).
+
+2. Select the account created in the above steps.
+
+3. Open Account Details screen, and then click Change.
+
+![Change Password](Images/password1.png)
+
+4. On the Change Password screen, specify how the CPM will change the password, and then click Change. The CPM (Central Policy Manager) changes the password after the one minute interval that you specified in the ImmediateInterval parameter.
+
+![Change Password](Images/password2.png)
+
+### Verify that the password is changed successfully
+
+1. After one minute, in the Account Details Screen, click Show to display the new password.
+
+![Change Password](Images/showpass1.png)
+
+![Change Password](Images/showpass2.png)
+
+2. On the Activities screen, Click the CPM Change Password action to display details about the password change process.
+
+## Login to EDB Postgres Advanced Server with PSQL Client using the new Password
+
+The following steps demonstrate that user can login to the EDB Postgres Advanced Server with the changed password.
+
+1. Open PSQL client and login to EDB Postgres Advanced Server with the changed password. Login will be successful.
```

Comment:
For this I would say something like "Login to EDB Postgres Advanced Server psql environment with the changed password.  This example uses a PSQL client on Windows." or something to that effect.